tool/dactyl-config.yml

```diff
@@ -222,13 +222,13 @@ pages:
             - local
             - ripple.com
 
-    ## Escrow Tutorial is a work in progress
-    # -   md: tutorial-escrow.md
-    #     html: tutorial-escrow.html
-    #     category: Tutorials
-    #     targets:
-    #         - local
-    #         - ripple.com
+    # Escrow Tutorial is a work in progress
+    -   md: tutorial-escrow.md
+        html: tutorial-escrow.html
+        category: Tutorials
+        targets:
+            - local
```

Comment:
Is `local` a draft placeholder for your own purpose? If so, it probably should be pulled before merging.

Reply:
No, `local` is the default target which is used to build the XRP Ledger docs. It creates an internally-consistent set of docs you can open as files or host in a static webserver. It's enormously useful for testing and staging purposes, although we use the ripple.com target for deploying to the site. (That one maps from the local filenames to the equivalent links on the site.)